src/webgpu/api/validation/encoding/cmds/setBindGroup.spec.ts

```diff
@@ -198,6 +198,54 @@ g.test('dynamic_offsets_passed_but_not_expected')
     validateFinish(false);
   });
 
+g.test('empty_bind_group_layouts_requires_empty_bind_groups')
```

Comment:
This validation doesn't occur inside setBindGroup() so this isn't the right file for these tests. I think `pipeline_bind_group_compat.spec.ts` is the correct file. Similar to the other tests there, we also need to test render passes and render bundles in addition to compute passes.

Reply:
Ah, I see. I move this test to `pipeline_bind_group_compat.spec.ts`.